tests/src/JIT/Methodical/structs/systemvbringup/structpinvoketests.cs

```diff
@@ -0,0 +1,1021 @@
+
+        public struct S3
+        {
+            public int x;
+            public int y;
+            public double z;
+        }
+
+        public struct S4
+        {
+            public int x;
+            public float y;
+        }
+
+        public struct S5
+        {
+            public int x;
+            public double y;
+        }
+
+        public struct S6
+        {
+            public short x;
+            public short y;
+            public int z;
+            public int w;
+        }
+
+        public struct S7
+        {
+            public double x;
+            public int y;
+            public int z;
+        }
+
+        public struct S8
+        {
+            public double x;
+            public int y;
+        }
+
+        public struct S9
+        {
+            public int x;
+            public int y;
+            public float z;
+            public float w;
+        }
+
+        public struct S10
+        {
+            public byte a;
+            public byte b;
+            public byte c;
+            public byte d;
+            public byte e;
+            public byte f;
+            public byte g;
+            public byte h;
+        }
+        
+        public struct S11
+        {
+            public byte a;
+            public byte b;
+            public byte c;
+            public byte d;
+            public double e;
+        }
+
+        public struct S12
+        {
+            public byte a;
+            public byte b;
+            public byte c;
+            public byte d;
+            public byte e;
+            public byte f;
+            public byte g;
+            public byte h;
+            public long i;
+        }
+
+        public struct S13
+        {
+            public byte hasValue;
+            public int x;
+        }
+
+        public struct S14
+        {
+            public byte x;
+            public long y;
+        }
+
+        public struct S15
+        {
+            public byte a;
+            public byte b;
+            public byte c;
+            public byte d;
+            public byte e;
+            public byte f;
+            public byte g;
+            public byte h;
+            public byte i;
+        }
+
+        public struct S16
+        {
+            public byte x;
+            public short y;
+        }
+
+        public struct S17
+        {
+            public float x;
+            public float y;
+        }
+
+        public struct S18
+        {
+            public float x;
+            public int y;
+            public float z;
+        }
+
+        public struct S19
+        {
+            public int x;
+            public float y;
+            public int z;
+            public float w;
+        }
+
+        // struct that doesn't fit into registers
+        public struct S20
+        {
+            public long x;
+            public long y;
+            public long z;
+            public long w;
+        }
+/* These tests are not working on non Windows CoreCLR
```

Comment:
It is a known problem that is on the list for interop team to solve – feel free to ask @yizhang82  about the details.